src/world/footpath.c

```diff
@@ -1740,9 +1740,9 @@ void footpath_update_path_wide_flags(int x, int y)
 	footpath_clear_wide(x, y);
 	y -= 0x20;
 
-	if (!(x & 0xE0))
+	if (!(x & 0xFFE0))
 		return;
-	if (!(y & 0xE0))
+	if (!(y & 0xFFE0))
```

Comment:
This change removes one of the issues with wide pathing that i mentioned whereby you could end up with disconnected areas. In these pictures I show a before and after shot. I've coloured the wide paths in the unselected colour (grey). 

![Before](https://cloud.githubusercontent.com/assets/1277401/17829428/6cb46cc8-66a6-11e6-9ed3-270f2f6c583d.PNG)

![After](https://cloud.githubusercontent.com/assets/1277401/17829430/81ad5e1e-66a6-11e6-9250-2d4541dba90c.PNG)

Note how the lower path in the before does not connect at all to the upper path.

This does mean that the wide paths no longer have the extra variety that the existing system has but as a bonus they are more likely to actually allow pathing. I think I'm okay with this change.

One final note about this code by changing it to `0xFFE0` you are actually removing these lines completely `footpath_update_path_wide_flags` is only called by x, y values that are multiples of 32. Therefore i think it better if you remove these two if statements completely.